presets/mainnet/network.yml

```diff
@@ -50,6 +50,8 @@ fileDatabaseBatchSize: 100
 minProofSize: 0
 syncTimeout: 5m
 maxChildNamespaces: 100
+nonVotingUnfinalizedBlocksDuration: 0m
+votingUnfinalizedBlocksDuration: 10m
```

Comment:
When the node is voting - votingUnfinalizedBlocksDuration: 0m
Nonvoting node - nonVotingUnfinalizedBlocksDuration: 10m